src/components/Graph.js

```diff
@@ -322,7 +359,7 @@ export class Graph extends React.Component {
     const link = this.state.link
       .data(links, d => [d.source, d.target])
       .join('line')
-      .attr('class', 'link');
+      .attr('class', d => {return d.fault ? 'link fault' : 'link'});
```

Comment:
Could we also set custom fault class to nodes affected by an alert (nodes participating in fault trajectory)? For instance, set a thick red border around it - the fill color should remain the same.